api/xml/xml_test.go

```diff
@@ -336,6 +336,14 @@ func TestScenario10(t *testing.T) {
 	executeScenario(t, "test10")
 }
 
+// `test11` is a clone of `test8` with certain revisions.
+// `test11` is for:
+// * Having an applicant that has a valid U.S. Passport
+// * Having an applicant that has ForeignBorn citizenship
```

Comment:
```suggestion
// * Having an applicant that has ForeignBorn (aka USByBirthOutsideUS) citizenship and has a U.S. Passport
```
Just trying to add some more clarity because this is such an a confusing aspect. The test is for both of those characteristics simultaneously (i.e., you can't move one of those bullet points to a different scenario and retain test coverage). And technically the passport can be expired.